src/transformers/trainer.py

```diff
@@ -758,17 +759,22 @@ def _wrap_model(self, model, training=True):
 
     def train(
         self,
-        resume_from_checkpoint: Optional[str] = None,
+        resume_from_checkpoint: Optional[Union[str, bool]] = None,
         trial: Union["optuna.Trial", Dict[str, Any]] = None,
         **kwargs,
     ):
         """
         Main training entry point.
 
         Args:
-            resume_from_checkpoint (:obj:`str`, `optional`):
-                Local path to a saved checkpoint as saved by a previous instance of :class:`~transformers.Trainer`. If
-                present, training will resume from the model/optimizer/scheduler states loaded here.
+            resume_from_checkpoint (:obj:`str` or :obj:`bool`, `optional`):
+
+                    * :obj:`str`: Local path to a saved checkpoint as saved by a previous instance of
+                      :class:`~transformers.Trainer`.
+                    * :obj:`bool`: Load the last checkpoint in `args.output_dir` as saved by a previous instance of
+                      :class:`~transformers.Trainer`.
```

Comment:
I would phrase this as a sentence: If a string, local path.... If a bool and equals to :obj:`True`, load the last...